src/main/java/com/linkedin/replica/wall/commands/impl/EditPostCommand.java

```diff
@@ -43,7 +43,7 @@ public Object execute() throws IOException {
         }
 
         boolean response = dbHandler.editPost(request);
-        cacheHandler.editPost(postId,request);
```

Comment:
In case of editing posts, company posts also should be deleted from cache, right?